Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+LABEL image=go-translate-server
+
+# Get all the dependencies
+RUN apt update
+RUN apt install -y git autoconf automake libtool curl make g++ unzip
+RUN apt install -y wget git-lfs pkg-config software-properties-common
+RUN git lfs install
+
+# Install protobuf
+RUN cd \
+    && git clone https://github.com/google/protobuf.git \
+    && cd protobuf \
+    && git checkout tags/v3.11.0 \
+    && git submodule update --init --recursive \
+    && ./autogen.sh \
+    && ./configure \
+    && make \
+    && make install \
+    && ldconfig
+
+# Install go
+RUN cd \
+    && wget https://dl.google.com/go/go1.13.linux-amd64.tar.gz \
```

Comment:
If the image used was Ubuntu 20.04, Go 1.13 would just be one `apt install golang` away.

Reply:
Otherwise, `wget --https-only --secure-protocol=PFS ...`

Reply:
No longer relevant for `go`, but added `--https-only --secure-protocol=PFS` to some other `wget` lines.